src/context/ShortcutSearch.css

```diff
@@ -0,0 +1,15 @@
+:export {
```

Comment:
General Question: Why did this PR touch shortcut search?

Reply:
Removed and added to gitignore